### PR TITLE
[BTS-279] Themification: Toggle

### PR DESCRIPTION
**Pull request by rapahaeru**
https://jirasoftware.catho.com.br/browse/BTS-279

### CODE REVIEW

File: components/Toggle/Toggle.jsx [hunk 1]
```diff
@@ -40,10 +49,18 @@ const Switch = styled.div`
...
+      },
+    }) => `${neutral[100]};`};
     border-radius: 50%;
     display: flex;
-    color: ${Colors.BLACK[400]};
+    color: ${({
+      theme: {
+        colors: { neutral },
+      },
+    }) => `${neutral[500]};`};
```
Comment (ddsilva) [inline components/Toggle/Toggle.jsx]: WDYT of putting all style rules of this component, that depend on theme props in just one callback?

File: components/Toggle/Toggle.jsx [hunk 2]
```diff
@@ -37,13 +46,18 @@ const Switch = styled.div`
...
+        colors: { neutral },
+      },
+    }) => `
+      background-color: ${neutral[100]};
+      color: ${neutral[500]};
+    `};
     border-radius: 50%;
     display: flex;
-    color: ${Colors.BLACK[400]};
     font-size: 12px;
```
Comment (alan-oliv) [inline components/Toggle/Toggle.jsx]: This shouldn't be: `font-size: ${baseFontSize * 0.75}px`?
respecting.

File: components/Toggle/Toggle.jsx [hunk 3]
```diff
@@ -22,10 +23,18 @@ const Switch = styled.div`
...
+    theme: {
+      spacing: { xxlarge },
+    },
+  }) => `${xxlarge}px;`}
 
   &:before {
-    background-color: ${Colors.BLACK[400]};
+    background-color: ${({
+      theme: {
+        colors: { neutral },
```
Comment (ggdaltoso) [inline components/Toggle/Toggle.jsx]:
```suggestion
        colors: { neutral: { 500: neutral500 } },
```
@ddsilva

File: components/Toggle/Toggle.jsx [hunk 4]
```diff
@@ -37,14 +46,20 @@ const Switch = styled.div`
...
+      },
+    }) => `
+      background-color: ${neutral[100]};
+      color: ${neutral[500]};
+    `};
     border-radius: 50%;
     display: flex;
-    color: ${Colors.BLACK[400]};
-    font-size: 12px;
+    font-size: ${({ theme: { baseFontSize: baseFont } }) =>
```
Comment (allyssonsantos) [inline components/Toggle/Toggle.jsx]: What do you think to put the font-size inside the callback above?

**Pull request merged**